pkg/redirectpolicy/manager.go

```diff
@@ -499,10 +499,6 @@ func (rpm *Manager) getAndUpsertPolicySvcConfig(config *LRPConfig) error {
 
 	case svcFrontendNamedPorts:
 		// Get service frontends with the clusterIP and the policy config named ports.
```

Comment:
Could you fix the comment as well? We don't need to get the port names here.

Reply:
Thanks for checking the code!

Regarding the comment,

> Get service frontends with the clusterIP and the policy config named ports.

It somehow looks reasonable to me. In this case, _config.frontendMappings_ has already been initialized in _redirectpolicy.go_ with the named ports, so that we are actually "getting frontends with the clusterIP and the named ports" here.

Do you have any suggestions?

Reply:
Oh right, no need to modify the comment.